pom.xml

```diff
@@ -47,9 +47,14 @@
     <scala.binary.version>2.12</scala.binary.version>
     <scala.version>2.12.15</scala.version>
     <spark.major.version>3</spark.major.version>
-    <sparkbundle.version>3.2</sparkbundle.version>
+    <sparkbundle.version>3.4</sparkbundle.version>
     <spark.version>3.4.2</spark.version>
-    <sparkshim.artifactId>spark-sql-columnar-shims-spark32</sparkshim.artifactId>
+    <sparkshim.module.name>spark34</sparkshim.module.name>
+    <sparkshim.artifactId>spark-sql-columnar-shims-spark34</sparkshim.artifactId>
+    <iceberg.version>1.5.0</iceberg.version>
+    <delta.package.name>delta-core</delta.package.name>
```

Comment:
If run `mvn clean` without this line, `delta.package.name` is undefined and we get a invalid artifactId in delta dependency..
Because profile java-8 is activated and all profiles that are active by default are automatically deactivated.

Ref: https://maven.apache.org/guides/introduction/introduction-to-profiles.html#details-on-profile-activation